Jenkinsfile

```diff
@@ -31,7 +30,7 @@ common.main {
         timestamps {
 
             stage('checkout') {
-                deleteDir()
+                sh "sudo rm -rf ${WORKSPACE}/*"
```

Comment:
Fixes deleteDir() root permissions issue

Reply:
Will it work on all builders?

Reply:
worked on all the builders i used today ;-)  (a lot)